google-cloud-storage/conformance/README.md

```diff
@@ -15,3 +15,5 @@ The output directory is the same is the input directory, so after successful com
 ## Running the tests
 
 The conformance tests are included in the ordinary unit test suite. In the `google-cloud-storage` directory, run `rake test`.
+
```

Comment:
Does the emulator need to be launched separately in order to run the tests? If so worth noting how to do that.

Reply:
Yes, it does. Updated README to include that. Thanks.